examples/data-sources/powerscale_smb_share_settings/provider.tf

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright (c) 2024 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+terraform {
+  required_providers {
+    powerscale = {
+      source = "registry.terraform.io/dell/powerscale"
+    }
+  }
+}
+
+provider "powerscale" {
```

Comment:
Don't forget to change it back to default config

Reply:
changed default config